src/main/webapp/index.html

```diff
@@ -53,10 +53,14 @@ <h1>How does it work?</h1>
   </p>
   <p>
     <strong class="list-mark">3.</strong>
-    Download image with blurred faces.
+    If you want to make some blurred area visible (e.g. to blur all your friends in the photo, but not yourself), simply click on that area in preview image.
```

Comment:
Missing period at the end of the sentence.

Reply:
What period do you mean? This sentence for looks like this:
'If you want to make some blurred area visible (e.g. to blur all your friends in the photo, but not yourself), simply click on that area in preview image.'

Reply:
Ah, I see, the sentence didn't fit my screen, so I missed the end of it.

Reply:
I guess I could argue that such long text should be placed on multiple lines, but it's more of a Nit, rather than a request for a change.